### ISSUE TITLE
Crash on Startup

### ISSUE THREAD

**Issue by NatalieBee** [mezz/JustEnoughItems#2562]
Hello,

I'm trying to test the latest version of JEI (jei-1.17.1-8.2.0.35). I'm running with forge 37.0.22 in Minecraft 1.17.1 with no other mods present and I keep getting a crash before the forge loader even starts in a new window. No crash report is generated. I can't see any obvious errors but I'll link the debug log and latest logs for you. https://gist.github.com/NatalieBee/6a3cfb33b389a471a73621c5eee28038

Oh, and to confirm, it starts up just fine without the mod present.

Thanks for your time,
Nat

**Comment by SpinHarmony** [comment]
You’re using a really old Forge version. You should update to latest. It will likely fix the problem (and would still be a good idea regardless).

**Comment by NatalieBee** [comment]
You totally called it. It took a little bit of work but updating to the latest Forge version fixed the issue. For anyone else reading this, I also had to fix an issue with my JVM arguments that prevented Minecraft from starting with the latest Forge version ("Could not create the Java virtual machine" error). It seems that one of the arguments I was using wasn't compatible. When editing the installation, I pressed the Reset button next to the JVM arguments and that fixed it.
